staff/lab/last-lab

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import sys
+
+import ocflib.lab.stats
+from ocflib.infra.hosts import hostname_from_domain
+from tabulate import tabulate
+
+
+def hostname(s):
+    """Argparse custom type that appends ".ocf.berkeley.edu" to short hostnames."""
+    if '.' not in s:
+        return s + '.ocf.berkeley.edu'
+    else:
+        return s
+
+
+def get_credentials():
+    """Return dictionary of ocfstats-ro creds, if we can read them."""
+    return json.load(open('/etc/ocfstats-ro.json', 'r'))
+
+
+def format_row(r):
+    """Given a row from the `session` table, formats it for printing."""
+    return (r['user'], hostname_from_domain(r['host']), r['start'],
+            'Still logged in' if r['end'] is None else r['end'])
```

Comment:
I think this would benefit from being split across more lines, something like:

```
return (
    r['user''],
    hostname_from_domain(['host']),
    r['start'],
    'Still logged in' if r['end'] is None else r['end'],
)
```

I think it just makes the number of items returned clearer and also makes it easier to see how each one is formatted.